speed-dreams/source-2.2.3/src/modules/sound/snddefault/SoundInterface.cpp

```diff
@@ -17,6 +17,7 @@
  *                                                                         *
  ***************************************************************************/
 
+#include <algorithm>
```

Comment:
Don't know why this was added here, maybe it was a bug on CLion?

Reply:
This was needed for std::min and std::max. They don't exist for me. Accidentally creeped in this PR